.github/workflows/pytest.yml

```
@@ -26,10 +26,11 @@ jobs:
         os: [macOS-latest, ubuntu-latest, windows-latest]
         python-version: ["3.9", "3.10", "3.11", "3.12"]
         chemlib: [obabel, rdkit]
-        graphlib: [nx, gt]
+        graphlib: [nx, gt, all]
         exclude:
         # graph-tools does not work on Windows
         - {os: "windows-latest", graphlib: "gt"}
+        - {graphlib: "all", chemlib: "obabel"}
```

```suggestion
        - {graphlib: "all", chemlib: "obabel"}
        - {graphlib: "all", os: "windows-latest"}
```
CI is annoying because it's often trial-and-error. I think we need a second configuration excluding Windows when using all backends.

---

I already realized the error, I just pushed it!